examples/example-gcp-observability/README.md

```diff
@@ -0,0 +1,39 @@
+gRPC GCP Observability Example
+================
+
+The GCP Observability example consists of a Hello World client and a Hello World server instrumented for logs, metrics and tracing. 
+
+__Please refer to GCP observability user guide for setting up authorization to acess Google Cloud Platform with Google user credentials.__
```

Comment:
Can we provide a link to the UG? We have one for preview and it's not going to change for GA, right?

Reply:
I checked with Lisa, she is trying to get a new directory set up. In that case, UG link will be different than what we have now.

Reply:
Okay, we can go with the current link and update it whenever we have the updated link

Reply:
Updated and added current user guide link.

Reply:
Link is correct but the link text "GCP observability user guide" doesn't match it or is not even correct. The user guide says it is for "Microservices observability" - so we could at least say that. In reality the UG is for "gRPC GCP Observability" or "gRPC Microservices on GCP Observability" . "GCP Observability" is too broad.

Reply:
I removed the link, as the link is going to be changed when we release. 
The product name is "Microservices Observability", should i add "gRPC Microservices Observability" instead here to show connection to gRPC (while we do not have a link). Let me know what you think.

Reply:
If you removed the link, make sure the description is as close to what the actual UG title is (since users will have to search to find it). I think it is "Microservices Observability"  so that makes most sense.

Reply:
Done